### PR TITLE
Building mobile responsive hamburger menu

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Screenshot from 2022-12-19 17-38-13

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('change');
+  navsub.classList.toggle('nav-change');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n
+  .addEventListener('click', () => {
+    hamburger.classList.remove('change');
+    navsub.classList.remove('nav-change');
+  }));
```
Comment (Qoosim) [inline index.js]: - [ ] It is required to stick to the Figma design as close as possible. The background color of your design is different from the guiding template as well as the opacity. The arrangement of the mobile menu options is not tally with what we have on the guiding template - this involves font-size, font-family, margin, padding, among others. Kindly look at the Figma design very closely and work your design alongside the guiding template. Please, take a look at both designs and draw the difference. 

### Your Design
your-design

### Figma Design
figma-design

Comment (Caren-Koroeny) [reply index.js:12]: Hello @Qoosim kindly where is the issue? I have not understood, because I used the figma design, is it the pop up menu?

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Caren-Koroeny** [comment]
- Thank you for the feedback, apologies for our ignorance, we had not seen this part on figma, we revisited it and saw that there are changes needed, a question though, does the menu have to cover the whole page when the user clicks on the hamburger menu?Thank you already.

**Comment by Caren-Koroeny** [comment]
- Thank you for the feed back, we worked on it.

**Review by Munsa1 — APPROVED** [review]
Hi @Caren-Koroeny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [ ] I would advise you to fix the overflow on the mobile version of your app as this will improve user experience and the quality of your app.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**